test/help/help.test.js

```diff
@@ -37,6 +37,7 @@ describe('help', () => {
         expect(pureStdout).toContain("To see list of all supported commands and options run 'webpack --help=verbose'.");
         expect(pureStdout).toContain('CLI documentation: https://webpack.js.org/api/cli/.');
         expect(pureStdout).toContain('Made with ♥ by the webpack team.');
+        expect(stdout).toMatchSnapshot();
```

Comment:
This is actually a bit tricky because `weback-4` and `webpack-5` have different snapshots. for ex `--target <value>` and `--target <value...>`.

Reply:
yeah need to come up with different strategy